python/cuml/test/dask/test_umap.py

```diff
@@ -75,7 +80,8 @@ def umap_mnmg_trustworthiness():
             return trustworthiness(X_transform, embedding, n_neighbors)
 
         def local_umap_trustworthiness():
-            local_model = UMAP(n_neighbors=n_neighbors)
+            local_model = UMAP(n_neighbors=n_neighbors,
+                               random_state=42)
             local_model.fit(local_X, local_y)
             embedding = local_model.transform(local_X)
             return trustworthiness(local_X, embedding, n_neighbors)
```

Comment:
Setting a `random_state` value probably changed some of the trustworthiness scores. Would be interesting to update the assertions accordingly.

Reply:
I intend to update these before the PR is merged, but I would like to see the CI results over a few different runs first.